docs/docs/troubleshooting/LIMITATIONS.md

```diff
@@ -24,11 +24,3 @@ Amazon Web Services (AWS) is the only supported cloud platform.
 ### Database
 
 PostgreSQL is the defined database engine for the products and cannot be modified in the configuration. However, users can change the database [instance type](../userguide/configuration/CONFIGURATION.md#database-instance-class) and [storage size](../userguide/configuration/CONFIGURATION.md#database-allocated-storage).
-
-### Scaling EKS
-
-You cannot change the number of the EKS cluster nodes (`desired_capacity`) and node type (`instance_types`) after provisioning the environment.
-
-### Scaling DC product
```

Comment:
We may can refer to this link for an alternative method to scale the product, without keeping the state.

Reply:
Not sure we should offer this as alternative, so far terraform is not really smart about drifts in helm

Reply:
Terraform is not involved if one using Helm chart to scaling the product directly. But I can understand if we don't want to provide any suggestion to scale out of our key-turn solution. Anyhow, if we scale the Helm chart without running terraform, terraform does not update the state and running any update (install.sh) in the environment will override the scaling.